pkg/compose/publish_test.go

```diff
@@ -23,7 +23,6 @@ import (
 
 	"github.com/compose-spec/compose-go/v2/loader"
 	"github.com/compose-spec/compose-go/v2/types"
-	"github.com/docker/compose/v2/internal/ocipush"
 	"github.com/docker/compose/v2/pkg/api"
 	v1 "github.com/opencontainers/image-spec/specs-go/v1"
```

Comment:
(very unrelated) In our other repositories, we alias this import as either `ocispec` (or `ocispecs`); I can open a follow-up PR to update the aliases in this repo, which may help reading the code.